build.js

```diff
@@ -178,12 +178,21 @@ function handle(options, compiler) {
   var skeleton = fs.readFileSync(__dirname + path.sep + options.skeleton_file, 'utf-8');
   var html = dataToHtml(skeleton, options.browsers, options.tests, options.compiler);
 
+  var createIterableObject = [];
+  try {
+    var compiled = options.compiler(
+      'global.__createIterableObject = function(a, b, c) { return function*() { yield a; yield b; yield c; }(); };'
```

Comment:
@webbedspace `Symbol.iterator` fallback already exists [in HTML](https://github.com/kangax/compat-table/pull/426/files#diff-d79397253b80964bde92888b84a6e368R20).
This added code is the fallback for a compiler that has no polyfill for generator function or `Symbol.iterator` but can compile generator function.

The fallback process is:
1. Generator function polyfill (but I think any compilers cannot polyfill it in eval)
2. `Symbol.iterator` polyfill
3. Generator function compilation (new!)

I can add No.4 "`Symbol.iteraotr` compilation", but IMO it is not great important.
Because almost compilers don't compile `Symbol.iterator`, but polyfill it.